addons/mail/static/src/xml/discuss.xml

```diff
@@ -135,7 +135,7 @@
         </t>
         <t t-if="!disableAddThread || !empty">
             <div class="o_mail_sidebar_title">
-                <h4 t-att-class="type == 'multi_user_channel' ? 'o_mail_open_channels' : ''">
+                <h4 t-att-class="type == 'multi_user_channel' ? 'o_mail_open_channels' : '' + 'd-flex flex-row justify-content-start align-items-stretch'">
```

Comment:
Drop this commit and the icon for Facebook.

It's not aligned with other channel names and looks strange.